.github/workflows/antora-build.yml

```diff
@@ -17,16 +17,60 @@ permissions:
   id-token: write
 jobs:
   build:
+    name: Build OpenMATERIAL documentation and deploy it to gh-pages
     runs-on: ubuntu-latest
+    services:
+      kroki:
+        image: yuzutech/kroki:0.15.1
+        env:
+          KROKI_MAX_URI_LENGTH: 8000
+          KROKI_BLOCKDIAG_HOST: blockdiag
+          KROKI_MERMAID_HOST: mermaid
+      blockdiag:
+        image: yuzutech/kroki-blockdiag:0.15.1
+      mermaid:
+        image: yuzutech/kroki-mermaid:0.15.1
+
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     steps:
-    - name: Checkout repository
+    - name: Checkout with submodules
       uses: actions/checkout@v4
+      with:
+         path: repo
+         fetch-depth: 0 # test if depth in fetch history is required
+         submodules: recursive
+
+    - name: Checkout documentation
+      if: ${{ github.event_name == 'repository_dispatch' && github.event.client_payload.head_ref }}
+      uses: actions/checkout@v4
+      with:
+        repository: martinfiebig/OpenMATERIAL
```

Comment:
Why is there a private repository checked out? Is this from a testing branch?

Reply:
Yes, this was a leftover from my local testing branch, I've removed the private repository.